### ISSUE TITLE
delora is super slow, compared to any lora variants (except randlora)

### ISSUE THREAD

**Issue by hhnqqq** [ExplainableML/DeLoRA#2]
Have you tried to optimize the training speed

**Comment by mwbini** [comment]
Hi, have you tried increasing the learning rate? DeLoRA needs 10-100x learning rate compared to typical LoRA variants (check appendix B in the paper for value ranges)
